c-lightning_with_postgresql_reliability.md

```diff
@@ -234,23 +238,32 @@ full_page_writes = on                   # gbd Ven  7 Feb 2020 10:58:33 CET https
 #### Setting Up a Standby Server for [streaming replication]
 
 ##### Make a first backup of the master on the standby server
+For your standby server to be able to track changes from the master it has to be put into the same initial state. This is done by performing a **base backup** from the master to the standby. Switch to the standby server to perform this.
 
+First, stop the server. This is necessary to use the slot during the first backup.
+```bash
+sudo systemctl stop postgresql
+```
+Make a safety copy of the old directory in case anything goes wrong:
+```bash
+mv /var/lib/postgresql/12/main/ /var/lib/postgresql/12/main.backup
+```
+Now, we clone the master onto the standby server and grant access to the postgres user.
+
+```bash
+pg_basebackup -h 192.168.0.5 -U postgres -D /var/lib/postgresql/12/main/ -P --password --slot node_a_slot
+# enter the password for user postgres when prompted
+sudo chown -R postgres:postgres /var/lib/postgresql/12/main
+
+```
+In order to tell the server that it should take the role of standby we create the following signal file (which can remain empty).
 ```bash
```

Comment:
It seems well written. According to https://www.postgresql.org/docs/12/continuous-archiving.html#BACKUP-PITR-RECOVERY, you might want to pay attention to the pg_wal directory (point 2 and 5).